pyproject.toml

```diff
@@ -7,6 +7,7 @@ ignore = ["C901", "D100", "D101", "D102", "D103", "W503", "W504"]
 exclude = ['.eggs', '.git', '.tox', '.venv', '.build', 'build', 'report', 'docs']
 max-line-length = 99
 max-complexity = 10
+extend-ignore = ["E203", "E704"]
```

Comment:
Lint was not passing with the recent changes on [black](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#configuration). That is why this config was necessary